products/apache-apisix.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Apache Apisix
+category: app
+permalink: /apisix
+alternate_urls:
+-   /apache-apisix
```

Comment:
```suggestion
permalink: /apache-apisix
alternate_urls:
-   /apisix
```

The main URL for other Apache product is usually (but not always unfortunately) apache-xxx.